index.html

```diff
@@ -209,6 +209,29 @@ <h2>Methods</h2>
       and its privacy and security properties for comparison to other methods.
     </p>
 
+    <section class="informative">
+      <h3>Method Reuse is Encouraged</h3>
+      <p>
+        While many forms of decentralization are desired, some convergence upon
+        widely useful DID methods is considered beneficial.
+        If you are considering defining a new DID method, consider first
+        surveying existing defined DID methods to see if there may be an
+        already existing method that may meet your needs. Many DID method
+        specifications can be found listed in the [[DID-SPEC-REGISTRIES]]. Some
+        DID methods are evaluated against various criteria in the
```

Comment:
```suggestion
        specifications can be found listed in the [[DID-SPEC-REGISTRIES]]. Some
        DID methods have been evaluated against various criteria in the
```

Reply:
Applied in 8f33538a30b43c16a4634cd64669cc8439820e67